packages/next/src/server-build.ts

```diff
@@ -948,10 +948,7 @@ export async function serverBuild({
     const appRouterStreamingActionLambdaGroups: LambdaGroup[] = [];
 
     for (const group of appRouterLambdaGroups) {
-      if (!group.isPrerenders || group.isExperimentalPPR) {
-        group.isStreaming = true;
-      }
-
+      group.isStreaming = true;
```

Comment:
We probably want to group this under the same flag/env we're using upstream to allow this

Reply:
Discussed offline, this is fine